### ISSUE TITLE
The About Us page's image is not visible

### ISSUE THREAD

**Issue by tenzintsongi** [Adityaarai/VROOM-Car-Rental#105]
When user views the "About Us" page, the image is not visible but the alt text shows that there is an image.

**Comment by Abrahamhyolmo** [comment]
i have fixed this issue by making some changes 
![Screenshot 2024-05-19 133617](https://github.com/Adityaarai/VROOM-Car-Rental/assets/119650143/afca4d36-8a29-44d2-98f0-30fd75b154d5)
![Uploading Screenshot 2024-05-19 133646.png…]()

**Comment by Abrahamhyolmo** [comment]
i have fixed this issue by making some changes
![image](https://github.com/Adityaarai/VROOM-Car-Rental/assets/119650143/18ef199e-752f-4525-952b-6223923be3f6)
<div class="profile-box">
            <h4>Prajwol Rai</h4>
            <small>Project Manager </small>
            <img src="{% static 'img/Baje.jpg' %}">
            <div class="social-box">
              <i class="fa-brands fa-facebook"></i>
              <i class="fa-brands fa-twitter"></i>
              <i class="fa-brands fa-linkedin"></i>
              <i class="fa-brands fa-instagram"></i>
            </div>
